tdrs-backend/tdpservice/vault/vault_client.py

```diff
@@ -0,0 +1,134 @@
+import hvac
+import os
+import sys
+
+
+class VaultClient:
```

Comment:
Need to add the followings:
1) Add caching for creds. Then need to add token refresh.
2) We will be adding more vault secret paths for each service/app, then need to add a wrapper function that gets secrets based on the paths

Reply:
Let's create the ticket then decide when they are needed